src/utils/auth.js

```diff
@@ -16,12 +18,12 @@ export const logout = () => {
 
   const onSuccess = ({ data }) => {
     localStorage.removeItem('session')
-    console.log('successfully logged out')
-    window.location.reload()
+    toast.success('successfully logged out')
+    window.location.replace('/login')
   }
 
   const onFailure = (error) => {
-    console.log('Logout request failed.' + error && error.response)
+    toast.error('Logout request failed.' + error && error.response)
```

Comment:
We don't want to tell the user that the logout request failed. it's better we don't log anything to console. but we shall need to collect this event for monitoring.